indico/modules/events/editing/fields.py

```diff
@@ -19,8 +19,9 @@
 
 
 class EditingFilesField(Dict):
-    def __init__(self, event, allow_claimed_files=False, **kwargs):
+    def __init__(self, event, contrib, allow_claimed_files=False, **kwargs):
```

Comment:
not sure if it's better to pass the contrib or just the code.. i guess the contrib is more future-proof.